contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -148,6 +157,7 @@ contract AlignedLayerServiceManager is
             batchesState[batchMerkleRoot].batcherAddress
         ] -= txCost;
         payable(msg.sender).transfer(txCost);
+        emit TaskResponseProcessed(batchMerkleRoot, msg.sender);
```

Comment:
isn't this almost the same as previous `emit BatchVerified(batchMerkleRoot);` event?
I think this event is a bit redundant.

Reply:
You are right,  solved in [8dea4d6](https://github.com/yetanotherco/aligned_layer/pull/840/commits/8dea4d6b4c9b6046de9af200fd103296e641758b)